admin/packaging.py

```diff
@@ -676,6 +676,10 @@ def __str__(self):
         "netaddr/eui/oui.txt",
         "package-contains-timestamped-gzip",
         "systemd-service-file-outside-lib",
+
+        # The binaries in ManyLinux wheel files are not compiled using Debian
+        # compile flags (dpkg-buildflags  --export).
+        "hardening-no-relro",
```

Comment:
Is this something that we think is a fundamental issue with how we build flocker, something that is not worth the trouble to fix or something we intend to track and fix?

http://blog.isis.poly.edu/exploitation%20mitigation%20techniques/exploitation%20techniques/2011/06/02/relro-relocation-read-only/ seems to suggest that this makes flocker less able to mitigate exploitation.